ts/test/ts_project_test.bzl

```diff
@@ -13,6 +13,7 @@ def _dir_test_impl(ctx):
 
     # assert the inputs to the tsc action are what we expect
     action_inputs = target_under_test[OutputGroupInfo]._action_inputs.to_list()
+    action_inputs = sorted(action_inputs)
```

Comment:
The depset order should not be relied on and changed as a result of this fix. Most tests in this file already do this sorting.